js/content/store.js

```diff
@@ -3872,29 +3870,27 @@ let StoreFrontPageClass = (function(){
                 .add("featuredrecommended", ".home_cluster_ctn")
                 .add("trendingamongfriends", ".friends_recently_purchased")
                 .add("discoveryqueue", ".discovery_queue_ctn")
-                .add("curators", ".steam_curators_ctn")
-                .add("morecuratorrecommendations", ".apps_recommended_by_curators_ctn")
+                .add("curators", ".steam_curators_ctn", Localization.str.homepage_curators)
+                .add("morecuratorrecommendations", ".apps_recommended_by_curators_ctn", Localization.str.homepage_curators)
                 .add("fromdevelopersandpublishersthatyouknow", ".recommended_creators_ctn")
                 .add("popularvrgames", ".best_selling_vr_ctn")
                 .add("homepagetabs", ".tab_container", Localization.str.homepage_tabs)
-                .add("gamesstreamingnow", ".live_streams_ctn")
-                .add("updatesandoffers", ".marketingmessage_area")
+                .add("gamesstreamingnow", ".live_streams_ctn", "", true)
+                .add("updatesandoffers", ".marketingmessage_area", "", true)
+                .add("topnewreleases", ".top_new_releases", Localization.str.homepage_topnewreleases)
+                .add("steamlabs", ".labs_cluster")
                 .add("homepagesidebar", ".home_page_gutter", Localization.str.homepage_sidebar);
 
             if (specialoffers) customizer.add("specialoffers", specialoffers.parentElement);
             if (browsesteam) customizer.add("browsesteam", browsesteam.parentElement);
             if (recentlyupdated) customizer.add("recentlyupdated", recentlyupdated.parentElement);
             if (under) customizer.add("under", under.parentElement.parentElement);
 
-            let dynamicNodes = Array.from(document.querySelectorAll(".home_page_body_ctn .home_ctn:not(.esi-customizer)"));
-            for (let i = 0; i < dynamicNodes.length; ++i) {
-                let node = dynamicNodes[i];
-                if (node.querySelector(".esi-customizer") || node.style.display === "none") { continue; }
-
-                let headerNode = node.querySelector(".home_page_content > h2,.carousel_container > h2");
-                if (!headerNode) { continue; }
+            let dynamicNodes = document.querySelectorAll(".home_page_body_ctn .home_ctn:not(.esi-customizer), .home_pagecontent_ctn");
+            for (let node of dynamicNodes) {
+                if (node.closest(".esi-customizer") || node.querySelector(".esi-customizer") || node.style.display === "none") { continue; }
```

Comment:
Why are you using `closest`?
Can't this be achieved with a `not:` selector?

Reply:
I...forgot. Might need to test again when the next sale comes around.